microsoft-edge/privacy-whitepaper.md

```diff
@@ -299,6 +309,14 @@ Microsoft Edge offers Read aloud, which can read out loud the content of a webpa
+
+To improve Microsoft Edge, we're always learning from our users.  As part of this learning, some users may experience new functionality before it's made available to everyone.  To enable new functionality for randomly selected users, Microsoft Edge regularly sends required info about your operating system, channel, version, country or region, and other device configuration data to the Microsoft Edge configuration service.  This data is sent with a resettable identifier unique to your browser.  Data is transmitted to the service over HTTPS.  This data is used for receiving updates to enable new functionality, to keep Microsoft Edge up to date and performing properly, and to improve our products and services.  Additional controls and configurations are available for organizations.  [Learn more](https://go.microsoft.com/fwlink/?linkid=2112125) 
+
+As a user, you can't turn off these browser updates, but you can control whether your product usage data is sent to Microsoft by going to **edge://settings/privacy** and changing the settings under **Help improve Microsoft Edge**.  
+
+To understand how new functionality affects Microsoft Edge and Microsoft services, Microsoft Edge sends a resettable identifier unique to your browser and a functionality tag that encodes what new functionality was enabled to Microsoft Edge and Microsoft services.  This helps us build experiences and the best browser for everyone.  The functionality tag is not unique to your installation of Microsoft Edge and is shared across all Microsoft Edge installations that share the same set of functionality being enabled.  Microsoft Edge sends this information in a HTTPS header to only Microsoft services.  The browser does not send this header when you browse InPrivate or in Guest mode.  You can prevent this data from being sent by going to **edge://settings/privacy** and turning off **Improve Microsoft products by sending crash reports and data about how you use the browser**.  [Learn how to reset the identifier unique to your browser](#Diagnostic-data-about-how-you-use-the-browser).  
+
 ## Resolve navigation errors
 
 If Microsoft Edge detects SSL connection timeouts, certificate errors,or other network issues that might be caused by a captive portal (for example, a Wi-Fi network at a hotel or airport), Microsoft Edge will make a request to `http://edge.microsoft.com/captiveportal/generate_204` and check the response code.  If the request is redirected to another URL, Microsoft Edge will open that URL in a new tab, assuming that it's a sign-in page.  Requests to the captive portal detection page is a stateless service, and requests aren't logged and cookies aren't sent or saved.  On Windows platforms, Microsoft Edge will use a Windows captive portal service.  Otherwise, the Microsoft Edge captive portal service is used.  To turn this service off, go to **edge://settings/privacy** and turn off **Use a web service to help resolve navigation errors**.
```

Comment:
```suggestion
If Microsoft Edge detects SSL connection timeouts, certificate errors,or other network issues that may be caused by a captive portal \(for example, a Wi-Fi network at a hotel or airport\), Microsoft Edge makes a request to `http://edge.microsoft.com/captiveportal/generate_204` and checks the response code.  If the request is redirected to another URL, Microsoft Edge opens that URL in a new tab, assuming that it is a sign-in page.  Requests to the captive portal detection page is a stateless service, and requests aren't logged and cookies aren't sent or saved.  On Windows platforms, Microsoft Edge uses a Windows captive portal service.  Otherwise, the Microsoft Edge captive portal service is used.  To turn this service off, go to `edge://settings/privacy` and turn off the **Use a web service to help resolve navigation errors** setting.
```